### DISCUSSION TITLE
Scrolling Not Happening

### DISCUSSION THREAD

**Discussion by arpit314ad** [Budibase/budibase#1946] [Help] [answered]
Hi...

I am not able to scroll the page with the new version in the developer and preview pages.
Please suggest.

Arpit
![Scroll](https://user-images.githubusercontent.com/86228170/124868958-2ca9ee80-dfde-11eb-93b9-2bef6bcbfc0c.JPG)
![01Scroll](https://user-images.githubusercontent.com/86228170/124868974-316ea280-dfde-11eb-939e-d8830260081f.JPG)

**Comment by PGLange** [comment]
Hi @arpit314ad,
I'm having the same issue, a kind of a workaround is to set height to 100vh and overflow: scroll; in corresponding css. Hope this helps.

As long as your content won't grow >100vh this works well....

**Reply by arpit314ad** [reply]
Hi..

Thanks for the revert.  I tried the setting up as  suggested but it is not happening.
![02Scroll](https://user-images.githubusercontent.com/86228170/124872811-5b769380-dfe3-11eb-90a6-687535a451a4.JPG)

**Reply by PGLange** [reply]
Hi @arpit314ad,
sorry, I didn't mention to set height and css values to your Screen. 
This should work - it does in my App.
![closed1](https://user-images.githubusercontent.com/86222990/124873987-789d6900-dfc7-11eb-816f-8f04a77371dd.PNG)

**Comment by arpit314ad** [comment]
Hi...

I did the same but there is no effect on the preview page. Out put is also attached.


![03 Scroll](https://user-images.githubusercontent.com/86228170/124881640-3850e180-dfed-11eb-8883-331267ab86cc.JPG)
![04 Scroll](https://user-images.githubusercontent.com/86228170/124881784-5e768180-dfed-11eb-9cf9-a333f42e03d6.JPG)

**Comment by aptkingston** [comment, marked as answer]
Hi @arpit314ad and @PGLange. The reason this is happening in your apps is because they were created using a version of Budibase before we recently updated how layouts and navigation work. The easiest option is to update your apps to use the new layout and navigation, which will fix this and also bring other improvements, such as much better responsive design support. 

To upgrade to the new layout, just go to the "Layouts" section in the builder and create a new layout, and configure that layout to look the same as your existing one. Links are now handled as a setting on the layout which makes it much easier to build your navigation. Once your layout is ready, go through each of your screens and change the layout setting from the current layout to your new one. Then you can delete the old layout.
